cmd/utils/flags.go

```diff
@@ -273,6 +279,16 @@ var (
 		Usage:    "Manually specify the Verkle fork timestamp, overriding the bundled setting",
 		Category: flags.EthCategory,
 	}
+	OverrideProofInBlock = &cli.BoolFlag{
+		Name:     "override.blockproof",
+		Usage:    "Manually specify the proof-in-block setting",
```

Comment:
Set it to true by default so that the kurtosis commands don't get modified after merging this PR.

```suggestion
		Usage:    "Manually specify the proof-in-block setting",
		Value:    true,
```